src/components/FormMovieAdd/helpers.ts

```diff
@@ -0,0 +1,12 @@
+import {FormMovieAddDTO} from "../../models/FormMovieAddDTO";
+import {MovieDetailsDTO} from "../../models/MovieDetailsDTO";
+
+export const createFormModel = (movieDetails?: MovieDetailsDTO): FormMovieAddDTO => ({
+    title: movieDetails?.title ?? "",
+    movieUrl: "",
+    genre: movieDetails?.genres ?? [],
+    overview: movieDetails?.description ?? "",
+    runtime: movieDetails?.duration ?? "",
+    rating: String(movieDetails?.rating) ?? "",
```

Comment:
Here you would have `String(undefined) -> "undefined"`, so you can make at least a check:
`movieDetails?.rating ? String(movieDetails?.rating) : ""`